client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -239,31 +270,42 @@ export const PrescriptionLineEditForm: React.FC<
               alignItems="center"
               display="flex"
               flexDirection="row"
-              justifyContent="flex-start"
-              gap={1}
+              gap={5}
             >
               <Grid>
-                <InputLabel style={{ fontSize: 12 }}>
-                  {t('label.issue')}
-                </InputLabel>
-              </Grid>
-              <Grid>
-                <NumericTextInput
-                  autoFocus
-                  disabled={disabled}
-                  value={issueUnitQuantity}
-                  onChange={handleIssueQuantityChange}
-                  min={0}
-                  decimalLimit={2}
-                />
+                <Box display="flex" alignItems="center" gap={1}>
+                  <InputLabel sx={{ fontSize: 12 }}>
+                    {t('label.prescribed-quantity')}
+                  </InputLabel>
+                  <NumericTextInput
+                    autoFocus
+                    disabled={disabled}
+                    value={prescribedQuantity}
+                    onChange={handlePrescribedQuantityChange}
+                    min={0}
+                    decimalLimit={2}
+                  />
```

Comment:
So one thing I noticed is that when you move from Prescribed input field to Issued input, the "Issued" value is briefly changed to the Prescribed until it is re-allocated, like this:

![2025-02-03 14 20 53](https://github.com/user-attachments/assets/ff21c533-16ef-4bf2-ae0e-48b07eb630d9)

This is because the "change" method is actually being called "onBlur" as well (internally to the Numeric input). You can get around this just by passing a dummy "onBlur" function to override it:

```tsx
<NumericTextInput
    autoFocus
    disabled={disabled}
    value={prescribedQuantity}
    onChange={handlePrescribedQuantityChange}
    min={0}
    decimalLimit={2}
    onBlur={()=> {}}
  />
```